JavaAssignment/src/praveen/Swapwithoutvariable.java

```diff
@@ -0,0 +1,15 @@
+package praveen;
+
+public class Swapwithoutvariable {
+	static void swapNum(int a,int b){
```

Comment:
1) Add a comment before class and method like what are you going to achieve with this class. 
2) Format your code before committing using ctrl +a  and ctrl + shift +f
3) Remove extra space in the class.

Rest of the logic looks good to me.